src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala

```diff
@@ -24,6 +24,7 @@ import org.broadinstitute.dsde.rawls.model.AttributeUpdateOperations.{
 }
 import org.broadinstitute.dsde.rawls.model.WorkspaceACLJsonSupport._
 import org.broadinstitute.dsde.rawls.model._
+import org.joda.time.DateTime
```

Comment:
joda-time is deprecated in favor of the JDK's `java.time`; see https://www.joda.org/joda-time/. I know we still have joda-time elsewhere in Orch. But if you don't need it, could you use something else, like `Instant`?